hawkbit-mgmt/hawkbit-mgmt-server/README.md

```diff
@@ -24,37 +24,65 @@ run org.eclipse.hawkbit.app.mgmt.MgmtServerStart
 The Management API can be accessed via http://localhost:8080/rest/v1
 The root url http://localhost:8080 will redirect directly to the Swagger Management UI
 
-### Clustering (Experimental!!!)
+# Clustering (Experimental!!!)
+## Remote Events between micro-services
 
-The micro-service instances are configured to communicate via Spring Cloud Bus. You could run multiple instances of any
-micro-service but hawkbit-mgmt-server. Management server run some schedulers which shall not run simultaneously - e.g.
-auto assignment checker and rollouts executor. To run multiple management server instances you shall do some extensions
-of hawkbit to ensure that they wont run schedulers simultaneously or you shall configure all instances but one to do not
-run schedulers!
+The micro-service instances are configured to communicate via Spring Cloud Stream using message channels.
+You can run multiple instances of any micro-service, including those consuming events.
+However, the `hawkbit-mgmt-server` should typically be run as a single instance, as it schedules time-sensitive jobs such as auto-assignment checking and rollout execution.
+If multiple management server instances are needed, you must extend hawkBit to ensure that scheduled tasks do not run concurrently.
+Alternatively, configure all but one instance to disable scheduler execution.
 
-## Optional Protostuff for Spring cloud bus
+## Event Channel Types in Spring Cloud Stream
 
-The micro-service instances are configured to communicate via Spring Cloud Bus. Optionally, you could
+Remote events in hawkBit are distributed through **two distinct types of channels**:
+
+### 1. Fanout Event Channel
```

Comment:
shouldn't this be put into the /site content and here (and to other readme-s) to just have reference to there?